### PR TITLE
extlinux: mark boot partition as legacy bios bootable

### PR DESCRIPTION
**Pull request by dorileo**
Make sure extlinux is consistent with syslinux and mark the boot partition as legacy
bios bootable.

Signed-off-by: Leandro Dorileo <leandro.maciel.dorileo@intel.com>

### CODE REVIEW

File: src/bootloaders/extlinux.c [hunk 1]
```diff
@@ -35,33 +35,44 @@
...
                 extlinux_cmd = NULL;
         }
 
+        if (sgdisk_cmd) {
+                free(sgdisk_cmd);
+                sgdisk_cmd = NULL;
+        }
+
+        base_path = boot_manager_get_boot_dir((BootManager *)manager);
+        OOM_CHECK_RET(base_path, false);
```
Comment (bryteise) [inline src/bootloaders/extlinux.c]: Why was this moved?

**Comment by bryteise** [comment]
Change looks fine, just a random question I had while reviewing.

File: src/bootloaders/extlinux.c (hunk 1 above)
Comment (dorileo) [reply src/bootloaders/extlinux.c]: just a logic organization, keeping all these global checks up in the beginning - no more than stylistic organization.
